Add a technology refresh page (#26)
CHTC/Articles: 2022-08-31-Technology-Refresh.md

---
title: Technology Refresh

author: Christina Koch

publish_on:
  - chtc

type: news

canonical_url: "https://chtc.cs.wisc.edu/Technology-Refresh.html"

image:
    path: "https://raw.githubusercontent.com/CHTC/Articles/main/images/PATh_Facility_Preview.jpeg"
    alt: Image of Servers

description: |
    Thanks to the generous support of the Office of the Vice Chancellor for Research and Graduate 

>     Education with funding from the Wisconsin Alumni Research Foundation, we have been able 

Reviewer: ```suggestion
    Education with funding from the Wisconsin Alumni Research Foundation, CHTC has been able 
```

    to execute a major refresh of hardware.  This provided 207 new servers for our systems, 
    representing over 40,000 batch slots of computing capacity.
excerpt: |
    Thanks to the generous support of the Office of the Vice Chancellor for Research and Graduate

>     Education with funding from the Wisconsin Alumni Research Foundation, we have been able

Reviewer: ```suggestion
    Education with funding from the Wisconsin Alumni Research Foundation, CHTC has been able
```

    to execute a major refresh of hardware.  This provided 207 new servers for our systems,
    representing over 40,000 batch slots of computing capacity.

card_src: "https://raw.githubusercontent.com/CHTC/Articles/main/images/PATh_Facility_Preview.jpeg"
card_alt: Image of Servers
---

> Thanks to the generous support of the Office of the Vice Chancellor for Research and Graduate Education with funding from the Wisconsin Alumni Research Foundation, we have been able to execute a major refresh of hardware.  This provided 207 new servers for our systems, representing over 40,000 batch slots of computing capacity. Most of this hardware arrived over the summer and we have started adding them to CHTC systems.

Reviewer: ```suggestion
Thanks to the generous support of the Office of the Vice Chancellor for Research and Graduate Education with funding from the Wisconsin Alumni Research Foundation, CHTC has been able to execute a major refresh of hardware.  This provided 207 new servers for our systems, representing over 40,000 batch slots of computing capacity. Most of this hardware arrived over the summer and we have started adding them to CHTC systems.
```

Continue reading to learn more about the types of servers we are adding and how to access them.

### HTC System

On the HTC system, we are adding 167 servers of new capacity, representing 36,352 job slots and 40 high-end GPU cards.

> The new servers will be running CentOS Linux 8 – see our previous email and website page about how to test your jobs and

Reviewer: ```suggestion
The new servers will be running CentOS Linux 8 – CHTC users should see our website page about how to test your jobs and
```
